tests/featureWriters/cursFeatureWriter_test.py

```diff
@@ -295,3 +295,134 @@ def test_curs_feature_multiple_anchors_mixed(self, testufo):
+                feature curs {
+                    lookup curs_LTR {
+                        lookupflag IgnoreMarks;
+                        pos cursive a <anchor NULL> <anchor 100 200>;
+                        pos cursive b <anchor 0 200> <anchor 111 200>;
+                        pos cursive c <anchor 100 200> <anchor NULL>;
+                    } curs_LTR;
+
+                } curs;
+                """
+        )
+
+    def test_curs_feature_mixed_forced_direction(self, testufo):
+        testufo["a"].unicode = ord("a")
+        testufo["b"].unicode = ord("b")
+        testufo["c"].unicode = ord("c")
+
+        glyph = testufo.newGlyph("d")
+        glyph.unicode = ord("d")
+        glyph.appendAnchor({"name": "exit.RTL", "x": 110, "y": 210})
+
+        glyph = testufo.newGlyph("e")
+        glyph.unicode = ord("e")
+        glyph.appendAnchor({"name": "entry.RTL", "x": 10, "y": 210})
+        glyph.appendAnchor({"name": "exit.RTL", "x": 121, "y": 210})
+
+        glyph = testufo.newGlyph("f")
+        glyph.unicode = ord("f")
+        glyph.appendAnchor({"name": "entry.RTL", "x": 110, "y": 210})
+
+        glyph = testufo.newGlyph("alef")
+        glyph.unicode = 0x0627
+        glyph.appendAnchor({"name": "entry", "x": 100, "y": 200})
+
+        glyph = testufo.newGlyph("beh")
+        glyph.unicode = 0x0628
+        glyph.appendAnchor({"name": "entry", "x": 0, "y": 200})
+        glyph.appendAnchor({"name": "exit", "x": 111, "y": 200})
+
+        glyph = testufo.newGlyph("jeem")
+        glyph.unicode = 0x062C
+        glyph.appendAnchor({"name": "entry", "x": 100, "y": 200})
+
+        glyph = testufo.newGlyph("heh")
+        glyph.unicode = 0x0647
+        glyph.appendAnchor({"name": "entry.LTR", "x": 110, "y": 210})
+
+        glyph = testufo.newGlyph("waw")
+        glyph.unicode = 0x0648
+        glyph.appendAnchor({"name": "exit.LTR", "x": 10, "y": 210})
+        glyph.appendAnchor({"name": "exit.LTR", "x": 121, "y": 210})
+
+        glyph = testufo.newGlyph("zain")
+        glyph.unicode = 0x0632
+        glyph.appendAnchor({"name": "entry.LTR", "x": 110, "y": 210})
+
+        generated = self.writeFeatures(testufo)
+
+        assert str(generated) == dedent(
+            """\
+                feature curs {
+                    lookup curs_ltr {
+                        lookupflag IgnoreMarks;
+                        pos cursive a <anchor NULL> <anchor 100 200>;
+                        pos cursive b <anchor 0 200> <anchor 111 200>;
+                        pos cursive c <anchor 100 200> <anchor NULL>;
+                    } curs_ltr;
+
+                    lookup curs_rtl {
+                        lookupflag RightToLeft IgnoreMarks;
+                        pos cursive alef <anchor 100 200> <anchor NULL>;
+                        pos cursive beh <anchor 0 200> <anchor 111 200>;
+                        pos cursive jeem <anchor 100 200> <anchor NULL>;
+                    } curs_rtl;
+
+                    lookup curs_LTR {
```

Comment:
was it intentional that these lookups are named `curs_LTR` and `cust_RTL` (capitalized)? It's weird to see them right next to the other two lookups named `curs_ltr` and `curs_rtl`. Can we merge them? Or if not can we give them names that are more distinctive? The parser is probably fine with it, but for a human it may be confusing to see both.

Reply:
The lowercase `ltr` and `rtl` are suffixes we added, when the upper case is part of the abchor name, so if the anchor was `entry.foo.bar.LTR`, the lookup would be `curs_foo.bar.LTR`.

My first iteration of the code produced lookups named like `curs_LTR_ltr`, so I dropped the suffix we added. I can instead drop the `LTR` and `RTL` part from the lookup name, it does not matter much either way.

Reply:
>  it does not matter much either way.

I don't know, but I think having two lookups named `curs_RTL` and `curs_rtl` is very confusing. But aside from the name, do they actually need to be two distinct lookups?

Reply:
We are keeping each anchor pair is a separate lookup. We don’t merge anchor lookups, regardless of the direction. This is not a new development.

Reply:
I’d rather just follow GlyphsApp here, which seems to not merge the lookups.